webview/src/plots/components/PlotsContainer.tsx

```diff
@@ -31,12 +31,15 @@ export type BasicContainerProps = Pick<
 >
 
 export const SectionDescription = {
+  // "Trends"
   [Section.CHECKPOINT_PLOTS]:
-    'Linear plots based on data from the experiments table.',
+    'Real-time graphs based on metrics from the Experiments Table',
+  // "Images"
   [Section.COMPARISON_TABLE]:
-    'A table used to display image plots side by side.',
+    'Displays image plots side by side across experiments.',
+  // "Data Series"
   [Section.TEMPLATE_PLOTS]:
-    'JSON, YAML, CSV or TSV files visualized using Vega pre-defined or custom Vega-Lite templates.'
+    'Graphs of JSON, YAML, CSV, or TSV plot files, visualized using `dvc plots` templates'
 }
```

Comment:
This is the main file to review for accurate descriptions @shcheklein . The other page I just copy edited a little but it's the same content you created.